src/VisualStudio/Core/Def/Implementation/Workspace/SourceGeneratedFileManager.cs

```diff
@@ -104,7 +102,7 @@ public GeneratedFileDirectoryInfo(DocumentId documentId, Type generatorType)
             // The IVsRunningDocumentTable is a free-threaded VS service that allows fetching of the service and advising events
             // to be done without implicitly marshalling to the UI thread.
             _runningDocumentTable = _serviceProvider.GetService<SVsRunningDocumentTable, IVsRunningDocumentTable>();
-            _runningDocumentTableEventTracker = new RunningDocumentTableEventTracker(
+            _ = new RunningDocumentTableEventTracker(
```

Comment:
I can't say I'm a particular fan here that we're creating a type with a side effect and then hiding it. This might be tricky to debug later -- can we just keep the field in this case?